frontend/src/components/ingress/Details.tsx

```diff
@@ -36,6 +37,29 @@ export default function IngressDetails() {
     );
   }
 
+  /** this function is used to format the url to be used in the Link component */
+  function isFormatURL(url: any) {
```

Comment:
The name of the function seems like it returns a boolean, but it returns a component. So it should be called something like LinkOrString or something like that.